subprojects/diagnostics/src/main/java/org/gradle/api/tasks/diagnostics/PropertyReportTask.java

```diff
@@ -30,6 +31,12 @@
 @DisableCachingByDefault(because = "Not worth caching")
 public class PropertyReportTask extends ProjectBasedReportTask {
     private PropertyReportRenderer renderer = new PropertyReportRenderer();
+    private String property = null;
```

Comment:
❌ The property should be of type `Property`

Reply:
Do you have an example of where this has been done before? Looking at a few tasks, I only see `String`s in use

Reply:
Here's an example: https://github.com/gradle/gradle/blob/9af99bed573e23bf742cc3efc6cc15a5d4030528/subprojects/build-init/src/main/java/org/gradle/buildinit/tasks/InitBuild.java#L81